u-boot_v2018.05-rc1/cmd/fat.c

```diff
@@ -53,6 +53,58 @@ U_BOOT_CMD(
 	"      be printed and performance will suffer for the load."
 );
 
+int do_firmware_load(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
+{
+	__u8 number __aligned(ARCH_DMA_MINALIGN);
+	char number_addr_str[32];
+	char *argv_internal[5];
+
+	if (argv[1] == NULL) {
+		printf("miss addr\n");
+		return 1;
+	}
+
+	argv_internal[1] = "mmc";
+
+#ifdef arm_vexpress_a9
+	argv_internal[2] = "0:0";
+#else
+	argv_internal[2] = "0";
+#endif
+
+	snprintf(number_addr_str, 32, "%p" , &number);
+	argv_internal[3] = number_addr_str;
+
+	argv_internal[4] = "firmware_selected";
+
+	printf("fatload %s %s %s %s\n", argv_internal[1], argv_internal[2], argv_internal[3], argv_internal[4]);
+
+	if (do_load(cmdtp, flag, 5, argv_internal, FS_TYPE_FAT) == 0) {
+
+		snprintf(number_addr_str, 32, "%p" , &number);
```

Comment:
Why do you add this line?

Reply:
We want to see the actual command called.